assets/app/view/game/merge.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require 'view/game/actionable'
+require 'view/game/undo_and_pass'
+
+module View
+  module Game
+    class Merge < Snabberb::Component
+      include Actionable
+
+      def render
+        corporation = @game.mergable_candidates.first
+        merge = lambda do
+          process_action(Engine::Action::Merge.new(
+            corporation.player ? @game.current_entity : @game.ndm,
+            corporation: corporation,
+          ))
+        end
+
+        children = [h(:div, "Merge candidates: #{@game.mergable_candidates.map(&:name)}")]
+        children << render_description(corporation)
+        children << h(:div, [h(:button, { on: { click: merge } }, 'Merge into NdM')])
```

Comment:
Should be generic and have NdM as a const.

Reply:
That's an idea. I need to make @game.ndm generic as well. Probably add a method to base that g_18_mex.rb can override?  What do you call the one that you merge into? Mergee for the one that is merged into the... Merger?

Or should this view be only used by 18MEX? Can it be put in a Module in that case?

Reply:
the views shouldn't have any game specific logic like this

Reply:
Ok, I have now generalized the view so it can be used by other titles.